boards/arm/xenvm/doc/index.rst

```diff
@@ -0,0 +1,171 @@
+.. xenvm:
+
+ARMv8 Xen Virtual Machine
+#########################
+
+Overview
+********
+
+This board allows to run Zephyr as Xen guest on any ARMv8 board that supports
+ARM Virtualization Extensions.
+
+.. figure:: xen_project_logo.png
+   :align: center
+   :alt: XenVM
+
+   Xen virtual Guest (Credit: Xen Project)
+
+It provides minimal set of devices:
+
+* ARM Generic timer
+* GICv2
```

Comment:
why not GICv3?

Reply:
Because GICv2 is supported on all platforms.

Reply:
Not really. The GICv3 specification doesn't mandate support for GICv2.

Reply:
Oh, didn't knew that. 

Anyways,  I tested this PR with on RCAR Gen3 hardware, which have GICv2 and system timer frequency of 8.320 MHz. Hence the default values.